communication.proto

```diff
@@ -35,8 +35,6 @@ message InnerMessage {
         BlockTxHashes BlockTxHashes = 13;
         BlockTxHashesReq BlockTxHashesReq = 14;
 
-        VerifyTxReq VerifyTxReq = 15;
```

Comment:
Please add this back, refactor this later.

Reply:
This line is useless.
`VerifyTxReq` can be removed from this `oneof`(`enum`) struct.